bundle/Resources/public/js/views/cof-contentcreationview.js

```diff
@@ -192,11 +191,12 @@ YUI.add('cof-contentcreationview', function (Y) {
          */
         _renderContentTypeSelector: function (event) {
             var selectorContainer = this.get('container').one(SELECTOR_CONTENT_TYPE),
-                contentTypeSelectorView = this.get('contentTypeSelectorView');
+                contentTypeSelectorView = this.get('contentTypeSelectorView'),
+                tooltip = Y.Node.create('<div class="cof-content-creation__tooltip cof-is-hidden"></div>');
```

Comment:
I'd avoid using YUI methods for it.
In this case the more preferable way is:
```javascript
tooltip = document.createElement('div');

tooltip.classList.add('cof-content-creation__tooltip');
tooltip.classList.add('cof-is-hidden');
```
This is faster, native way.